cucumber-core/src/main/java/io/cucumber/core/order/factory/PickleOrderFactory.java

```diff
@@ -0,0 +1,10 @@
+package io.cucumber.core.order.factory;
+
+import io.cucumber.core.order.PickleOrder;
+
+public interface PickleOrderFactory {
```

Comment:
This class is part of Cucumber public API. 

We are using the API Guardian annotations to document that. Please add  `@API(status = API.Status.EXPERIMENTAL)` to this class. Same for `PickleOrder`.